README.md

```diff
@@ -124,6 +132,10 @@ Short description.
 
   Specialized in electric car manufacturing and through, its SolarCity subsidiary, solar panel manufacturing.
 
+- [Tessol](https://www.tessol.in/) | Mumbai, India | 2013
```

Comment:
Not sure if this company classifies as clean-tech. 

From their website I cannot find anything related to sustainability. Their main goal seems to be `Redefining Cold Chain Logistics`, it might be that they end up making something with a good impact, which is great, but the main goal of the company doesn't seem to be related to improving the environment.

Reply:
Yes okay, I've researched and added another company instead. I will send a PR.

Thanks for reviewing my PR